packages/web-app-files/tests/unit/views/PrivateLink.spec.js

```diff
@@ -0,0 +1,16 @@
+describe('PrivateLink view', () => {
+  describe('when the page has loaded successfully', () => {
+    it.todo('should have the background image set')
+    it.todo('should display the page title')
+    it.todo('should display the logo')
+    it.todo('should call "getPathForFileId" method')
+  })
+
+  describe('when "loading" is set as true', () => {
```

Comment:
I think 
```suggestion
  describe('when the page is still loading', () => {
```
would be a better description for the group